src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return (cost - 50);
+  } else if (days >= 3) {
+    return (cost - 20);
+  } else {
+    return cost;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return (cost - 50);
  } 
  
  if (days >= 3) {
    return (cost - 20);
  }
  
   return cost;
```
![image](https://user-images.githubusercontent.com/96409563/205506976-97aab178-3005-4e8d-bfc4-b81b3807df0d.png)